Packs/Whois/Integrations/Whois/test_data/EU domains.text

```diff
@@ -0,0 +1,67 @@
+% The WHOIS service offered by EURid and the access to the records
```

Comment:
Do we need the whole file for testing?

Reply:
I think so,
The `parse_raw_whois` function gets an argument `raw_data` which is the content of the file.